syntax ="proto3";

service read_db {
    rpc read_db(read_db_msg) returns(response_msg){}
}

service write_db {
    rpc write_db(write_db_msg) returns(response_msg){}
}

message read_db_msg {
   string query = 1;
}

message write_db_msg {
   string query = 1;
}

message response_msg {
  string response = 1;
}
